libcontainer/container_linux.go

```diff
@@ -919,8 +919,19 @@ func (c *linuxContainer) criuSupportsExtNS(t configs.NamespaceType) bool {
 	return c.checkCriuVersion(minVersion) == nil
 }
 
+// title converts the first character of ASCII string s to uppercase.
+func title(s string) string {
+	s0 := s[0]
+	if s0 < 'a' || s0 > 'z' { // not lowercase ASCII
+		return s
+	}
+	s0 -= 'a' - 'A' // convert to uppercase
+
+	return string(s0) + s[1:]
+}
+
 func criuNsToKey(t configs.NamespaceType) string {
-	return "extRoot" + strings.Title(configs.NsName(t)) + "NS"
+	return "extRoot" + title(configs.NsName(t)) + "NS"
```

Comment:
Is this parsed by something?
Do we really need to titlize it?

Reply:
Quoting commit message

> The alternative is to not capitalize the namespace string; this will break
> restoring of a container checkpointed by earlier version of runc.

Reply:
I moved back to using `strings.Title` instead, suppressing the deprecation warning as it seems easier.